easystage1.js

```diff
@@ -0,0 +1,46 @@
+const qn = [
+  {question: 'A game arcade released in 1980', inputBox: 6, key: ''},
+  {question: 'One of the most best-selling video game of all time', inputBox: 9, key: 'Letters collected: n'},
+  {question: 'Play anywhere, anytime, with anyone', inputBox: 8, key: 'Letters collected: ne'},
+  {question: 'Name of the silent black cat that deputed in 1919', inputBox: 5, key: 'Letters collected: ned'},
+  {question: 'Name of a garden', inputBox: 4, key: 'Letters collected: nede'}]
+
```

Comment:
This feels like a very hardcoded way of doing things. Please update the google docs and ping me to add this to the server such that it returns you a new question + other contents. 

One way would be the person would send up the answer and the key from the previous question. In return, the server would return the letter (only one) for that specific key and the key for the next question